### PR TITLE
[Merged by Bors] - Bump go modules such as helm to v3.10.1 and k8s client to v0.25.3

### PR DESCRIPTION
**Pull request by sbernauer**
## Review Checklist
- [x] Code contains useful comments
- [x] (Integration-)Test cases added (or not applicable)
- [x] Documentation added (or not applicable)
- [x] Changelog updated (or not applicable)
- [x] Cargo.toml only contains references to git tags (not specific commits or branches)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@
 ### Changed
 
 - Bump to clap 4.0 ([#150](https://github.com/stackabletech/stackablectl/pull/150))
+- Bump go modules such as helm to v3.10.1 and k8s client to v0.25.3 ([#148](https://github.com/stackabletech/stackablectl/pull/148))
```
Comment (siegfriedweber) [inline CHANGELOG.md:13]:
```suggestion
- Bump go modules such as helm to v3.10.1 and k8s client to v0.25.3 ([#149](https://github.com/stackabletech/stackablectl/pull/149))
```

**Review by siegfriedweber — CHANGES_REQUESTED** [review]
LGTM. Works on NixOS with Go 1.18.

**Comment by sbernauer** [comment]
Thanks for confirming!

**Review by siegfriedweber — APPROVED** [review]

**Comment by siegfriedweber** [comment]
bors merge

**Pull request closed without merging**